google-calendar.js

```diff
@@ -183,8 +183,24 @@ function addRow(event, counter, defaultTicket) {
     ticketMarkUp = '<input id="ticket[' + counter + ']" class="form-control" type="text" value="'+ ticket +'"">';
     ticketCell.innerHTML = ticketMarkUp;
   }
+  
+  // check if log is enabled and user attended the meeting and if they did, check the box next to the meeting
+  function shouldLog(event) {
+    if (checkIfLogEnabled(event) && hasAttended(event)) {
```

Comment:
When I test this code, I get an error saying that hasAttended is not defined. I believe this is trying to call the checkIfAttended() function?

https://cl.ly/0676eb358056

Reply:
You are right. It's fixed now. Thank you.